lib/index.js

```diff
@@ -24,6 +24,15 @@ class GoodLoggly extends Stream.Writable {
       useTagHeader: false,
       tags: Array.isArray(config.tags) ? config.tags : []
     });
+
+    this._buffer = [];
+
+    // Send last messages on your way out
+    this.once('finish', () => {
+      if (this._buffer.length) {
```

Comment:
I'd also move this check into `_sendMessages()`, assuming that we don't want to allow empty arrays to be sent.